lib/logstash/agent.rb

```diff
@@ -50,6 +50,11 @@ class LogStash::Agent < Clamp::Command
     I18n.t("logstash.agent.flag.configtest"),
     :attribute_name => :config_test
 
+  option "--[no-]force-shutdown", :flag,
```

Comment:
+1

I would personally prefer `--unsafe-shutdown`. It makes it clear that it's a risk.